pkg/errors/wrap.go

```diff
@@ -0,0 +1,39 @@
+package errors
```

Comment:
added this to remove the need for the "errors" package.  Should be able to do everything we need without that package anymore.